kubernetes-model/openshift-model/src/main/java/io/fabric8/openshift/api/model/Template.java

```diff
@@ -61,7 +61,7 @@
 @JsonDeserialize(using = JsonDeserializer.None.class)
 @EqualsAndHashCode
 @ToString
-@Buildable(editableEnabled = false, validationEnabled = false, generateBuilderPackage=true, builderPackage = "io.fabric8.kubernetes.api.builder", inline = @Inline(type = Doneable.class, prefix = "Doneable", value = "done"))
+@Buildable(editableEnabled = false, validationEnabled = false, generateBuilderPackage=false, builderPackage = "io.fabric8.kubernetes.api.builder", inline = @Inline(type = Doneable.class, prefix = "Doneable", value = "done"))
```

Comment:
Why is this changed to false? Class generated with Annotator has it true

Reply:
See https://github.com/fabric8io/kubernetes-client/pull/2137#discussion_r410024370